.gitignore

```diff
@@ -0,0 +1,7 @@
+robot-venv
+provisioning/tmp/*
+*.xml
+*.html
+*.log
+*.pyc
+*.srl
```

Comment:
No new line at the end of the file. Please, add it

Reply:
added